src/header.rs

```diff
@@ -124,6 +124,12 @@ pub struct GnuHeader {
     pub pad: [u8; 17],
 }
 
+/// Description of a spare entry.
+pub struct SparseEntry {
+    pub offset: u64,
```

Comment:
Let's also document the fields please

Reply:
Offset and size names are self explanatory.